openquake/baselib/hdf5.py

```diff
@@ -343,8 +344,12 @@ def save(self, nodedict, root=''):
         attrib = nodedict.get('attrib', {})
         path = '/'.join([root, tag])
         nodes = nodedict.get('nodes', [])
-        if text:
-            setitem(path, text)
+        if text not in ('', None):
```

Comment:
Please add a comment about the fact that text could be a number, so `0` should not be discarded